libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -37,8 +38,14 @@ internal class RouteRefreshController(
 
     private var requestId: Long? = null
 
+    var routeRefreshInterval = routeRefreshOptions.intervalMillis
+        set(value) {
+            field = value
+            routerRefreshTimer.restartAfterMillis = routeRefreshInterval
+        }
+
     init {
-        routerRefreshTimer.restartAfterMillis = TimeUnit.MINUTES.toMillis(5)
+        routerRefreshTimer.restartAfterMillis = routeRefreshInterval
```

Comment:
Could you add a unit test for this in the `RouteRefreshControllerTest`?

Reply:
Yep, good call. Updated an existing test and added [should refresh route according to options](https://github.com/mapbox/mapbox-navigation-android/pull/4321/files#diff-f9cd0342c0c73c28b4e1998bdb1be641e387915f52a9e873f0089cdff15bd962R77)